dotcom-rendering/src/components/marketing/banners/designableBanner/DesignableBannerV2.tsx

```diff
@@ -0,0 +1,286 @@
+		containerSettings: {
+			backgroundColour: neutral[100],
+			textColor: neutral[0],
+		},
+		headerSettings: {
+			textColour: neutral[0],
+		},
+		primaryCtaSettings: {
+			default: {
+				backgroundColour: brandAlt[400],
+				textColour: neutral[0],
+			},
+			hover: {
+				backgroundColour: brandAlt[400],
+				textColour: neutral[0],
+			},
+		},
+		//not used in this design but is required to be passed in
+		secondaryCtaSettings: {
+			default: {
+				backgroundColour: brandAlt[400],
+				textColour: neutral[0],
+			},
+			hover: {
+				backgroundColour: brandAlt[400],
+				textColour: neutral[0],
+			},
+		},
+		closeButtonSettings: {
+			default: {
+				backgroundColour: neutral[100],
+				textColour: neutral[0],
+				border: `1px solid ${neutral[38]}`,
+			},
+			hover: {
+				backgroundColour: neutral[100],
+				textColour: neutral[0],
+				border: `1px solid ${neutral[100]}`,
+			},
+		},
+		highlightedTextSettings: {
+			textColour: neutral[0],
+			highlightColour: neutral[100], //set to be white as we may want this in the future?
+		},
+		articleCountTextColour: neutral[0],
+		bannerId: 'designable-banner',
+	};
+
+	const showAboveArticleCount =
+		(separateArticleCountSettings?.type === 'above' ||
+			separateArticleCount) &&
+		articleCounts.forTargetedWeeks >= 5;
+
+	return isTabletOrAbove ? (
+		<>
+			<div css={styles.guardianLogoContainer}>
+				<SvgGuardianLogo />
+			</div>
+		</>
+	) : (
+		<div
+			css={styles.outerContainer(
+				templateSettings.containerSettings.backgroundColour,
+				templateSettings.containerSettings.textColor,
+				iosAppBannerPresent,
+			)}
+		>
+			<div css={styles.containerOverrides}>
+				<DesignableBannerCloseButton
+					onCloseClick={onCloseClick}
+					settings={templateSettings.closeButtonSettings}
+					styleOverides={styles.closeButtonOverrides}
+				/>
+				<div>
+					<DesignableBannerHeader
+						heading={content.mainContent.heading}
+						mobileHeading={content.mobileContent.heading}
+						headerSettings={templateSettings.headerSettings}
+						headlineSize={design.fonts?.heading.size ?? 'medium'}
+					/>
+				</div>
+				<div>
+					{showAboveArticleCount && (
+						<DesignableBannerArticleCount
+							numArticles={articleCounts.forTargetedWeeks}
+							settings={templateSettings}
+							copy={separateArticleCountSettings?.copy}
+						/>
+					)}
+					<div>
+						<DesignableBannerBody
+							mainContent={content.mainContent}
+							mobileContent={content.mobileContent}
+							highlightedTextSettings={
+								templateSettings.highlightedTextSettings
+							}
+						/>
+					</div>
+				</div>
+				<div>
+					<ThreeTierChoiceCards
+						countryCode={countryCode}
+						selectedProduct={threeTierChoiceCardSelectedProduct}
+						setSelectedProduct={
+							setThreeTierChoiceCardSelectedProduct
+						}
+						variantOfChoiceCard={variantOfChoiceCard}
+					/>
+				</div>
+				<div css={styles.linkButtonContainer}>
+					<LinkButton
+						href={
```

Comment:
I'll re-look into using the Designable CTAs - I couldn't quite get it working initially so this is just temporary